test/LockedNORIUpgrade.test.ts

```diff
@@ -5,20 +5,20 @@ import type {
   LockedNORI,
   LockedNORIV1,
   MockERC777,
-  LockedNORIHelper,
+  LockedNORIV1Helper,
 } from '@/typechain-types';
 
 describe('LockedNORI V1 to V2 upgrade', () => {
   it('works before and after upgrading', async () => {
     const { bpNori, hre } = await setupTest();
-
-    const recipient = hre.namedAccounts.investor1;
+    const { namedSigners, namedAccounts } = hre;
+    const recipient = namedAccounts.investor1;
     const GRANT_AMOUNT = ethers.utils.parseEther('100');
 
     const helperFactory = await hre.ethers.getContractFactory(
-      'LockedNORIHelper'
+      'LockedNORIV1Helper'
```

Comment:
Use the generic instead to avoid casting in following lines (e.g., `const helper = `)
```ts
const helperFactory =
      await hre.ethers.getContractFactory<LockedNORIV1Helper__factory>(
        'LockedNORIHelper'
      );
```